### PR TITLE
ARTEMIS-1825 Live-backup topology not correctly displayed on console

### PR DESCRIPTION
**Pull request by gaohoward**

### CODE REVIEW

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js [hunk 1]
```diff
@@ -455,16 +457,27 @@ var ARTEMIS = (function(ARTEMIS) {
...
+                              var newBackup = newBackReq.value;
+                              if (!newBackup) {
+                                 ARTEMIS.log.info("yes I'm master right now");
+                                 if (remoteBroker.backup) {
+                                    getOrAddBroker(false, "\"" + remoteBroker.backup + "\"", remoteBroker.nodeID, "remote", null, properties);
+                                    addLinkIds("broker:" + broker.brokerId, "broker:" + "\"" + remoteBroker.backup + "\"", "network");
+                                 }
+                              }
+                              else {
+                                 ARTEMIS.log.info("ok I'm backup!");
```
Comment (clebertsuconic) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js]: debug message

Comment (gaohoward) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js]: Yes should be debug. I'll change that. Thx.

**Comment by gaohoward** [comment]
@clebertsuconic done.

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js [hunk 2]
```diff
@@ -461,10 +463,20 @@ var ARTEMIS = (function(ARTEMIS) {
...
                            }
                            else {
-                              var backup = remoteBroker.backup;
-                              if (backup) {
-                                 getOrAddBroker(false, "\"" + remoteBroker.backup + "\"", remoteBroker.nodeID, "remote", null, properties);
-                                 addLinkIds("broker:" + broker.brokerId, "broker:" + "\"" + remoteBroker.backup + "\"", "network");
+                              var newBackReq = ARTEMISService.artemisConsole.isBackup(jolokia, mBean);
+                              var newBackup = newBackReq.value;
+                              if (!newBackup) {
+                                 ARTEMIS.log.debug("yes I'm master right now");
```
Comment (clebertsuconic) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/brokerDiagram.js]: You still have unintended debug messages here

**Comment by gaohoward** [comment]
@clebertsuconic removed the debug logs.

**Comment by clebertsuconic** [comment]
@mtaylor / @andytaylor  / @michaelandrepearce  I didn't work on this module a lot... can you guys check on this?

**Comment by michaelandrepearce** [comment]
@gaohoward sorry taken a while to look at this as have been burning hours on other issue. From what i can tell from the JS this looks ok, i assume form reading the js, the idea here is more clearly show who is live and who is a slave, calling a method to deduce this, rather than the hardcoded master=true, is this roughly a correct understanding? If so +1 from me.

**Comment by gaohoward** [comment]
@michaelandrepearce  Yes that's the idea.

**Comment by michaelandrepearce** [comment]
Great ill merge this shortly then. Thanks for the contribution, and sorry again for taking soo long to review it

**Comment by gaohoward** [comment]
np at all.

**Pull request closed without merging**